### PR TITLE
Questions for SQL & Viz Portion of the Exercise

### PR DESCRIPTION
**Pull request by msandt3**
SQL Exercise & Viz Portion

- Adds questions & backing data set for SQL exercise
- Instructions for visualization portion of the exercise.

### CODE REVIEW

File: sql_challenge/data/continent_map.csv [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+NER,AF
+NFK,OC
+NGA,AF
+NIC,NA
+NIU,OC
+NLD,EU
+NOR,EU
+NPL,AS
+NRU,OC
+,OC
```
Comment (sonny-mohnalkar) [inline sql_challenge/data/continent_map.csv]: I believe these 4 countries are null which will need to be displayed as N/A

Comment (msandt3) [reply sql_challenge/data/continent_map.csv]: correct

File: README.md [hunk 2]
```diff
@@ -14,7 +14,73 @@ The exercise consists of three parts.
...
+* Provide all code you used to answer questions, set up the db, and import data in the [SQL Directory](/sql_challenge/sql)
+* Leave comments where you feel clarity is needed. 
+
+### Questions
+
+1. Data Integrity & Cleanup
+
+    Alphabetically list all the country codes in the continent map table that appear more than once. For countries with no country code make them display as "N/A" and display them first in the list. 
+
+2. List the Top 10 Countries by year over year % GDP per capita growth between 2011 & 2012.
```
Comment (sonny-mohnalkar) [inline README.md:37]: I see blank gdp_per_capita for Countries in the data set.  Do you feel necessary to specifically mention how we expect them to handle null gdp_per_capita to report the Top 10?

Comment (msandt3) [reply README.md:37]: I'd like to see them make an informed decision on how to handle missing values

File: README.md [hunk 3]
```diff
@@ -14,7 +14,73 @@ The exercise consists of three parts.
...
+    - Continent
+    - Growth Percent
+
+3. For the year 2012, compare the percentage share of GDP Per Capita for the following regions: North America (NA), Europe (EU), and the Rest of the World. Your result should look something like:
+
+    North America  | Europe | Rest of the World 
+    ------ | ------ | -------------
+    X%  | Y%  | Z%
+
+4. For years 2004 through 2012, calculate the average GDP Per Capita for every continent for every year.
```
Comment (sonny-mohnalkar) [inline README.md]: Is it necessary for define **Averge GDP per Capita per Contnent**?  I believe we are expecting it as
**sum of per_capita_gdp for all countries in a continent** / **number of countries in that continent**

File: README.md [hunk 4]
```diff
@@ -14,7 +14,73 @@ The exercise consists of three parts.
...
+    X%  | Y%  | Z%
+
+4. For years 2004 through 2012, calculate the average GDP Per Capita for every continent for every year. 
+
+    The final product should include columns for:
+    - Year
+    - Continent
+    - Average GDP Per Capita
+
+5. For years 2004 through 2012, calculate the median GDP Per Capita for every continent for every year.
```
Comment (sonny-mohnalkar) [inline README.md]: Below instructions for finding **Median** could help
To find the median, arrange the observations in ascending order. If there is an odd number of observations, the median is the middle value. If there is an even number of observations, the median is the average of the two middle values.

Comment (msandt3) [reply README.md]: I think so yes

**Review by sonny-mohnalkar — APPROVED** [review]

**Pull request merged**